webpack.config.js

```diff
@@ -97,20 +83,71 @@ const guiConfig = _.assign({
       return callback()
     }
   ],
+  module: {
+    rules: [
+      {
+        test: /\.jsx?$/,
+        include: [ path.resolve(__dirname, 'lib/gui') ],
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: [
+              '@babel/preset-react',
+              [ '@babel/preset-env', { targets: { electron: '3' } } ]
+            ],
+            plugins: [ '@babel/plugin-proposal-function-bind' ],
+            cacheDirectory: true
+          }
+        }
+      },
+      {
+        test: /\.html$/,
+        include: [ path.resolve(__dirname, 'lib/gui/app') ],
+        use: {
+          loader: 'html-loader'
+        }
+      },
+      {
+        test: /\.s(a|c)ss$/,
+        use: [
+          // eslint-disable-next-line no-negated-condition
+          process.env.NODE_ENV !== 'production' ? 'style-loader' : MiniCssExtractPlugin.loader,
+          // 'style-loader',
+          'css-loader',
+          'resolve-url-loader',
+          'sass-loader'
+        ]
+      },
+      {
+        test: /\.(eot|woff|woff2|svg|ttf)([?]?.*)$/,
+        use: [
+          {
+            loader: 'file-loader',
+            options: {
+              name: 'fonts/[name].[ext]'
+            }
+          }
+        ]
+      }
+    ]
+  },
   entry: {
     gui: path.join(__dirname, 'lib', 'gui', 'app', 'app.js')
   },
-  output: {
-    path: path.join(__dirname, 'generated'),
-    filename: '[name].js'
-  }
-}, commonConfig)
+  plugins: [
+    new MiniCssExtractPlugin({
+      filename: '[name].css',
+      chunkFilename: '[id].css'
+    })
+  ]
+})
 
-const etcherConfig = _.assign({
+const etcherConfig = _.assign(_.cloneDeep(commonConfig), {
```

Comment:
You can use `_.assign({}, commonConfig, { other: stuff })` and avoid the cloneDeep.